### PR TITLE
DOC: Clarify Om0 in Planck18 cosmology

### LINKED ISSUE #10957: cosmology.Planck18_arXiv_v2 parameter typo?
### Description

The cosmology.Planck18_arXiv_v2.Om0 value is 0.30966
But the value presented in Table 2 of the Planck2018 paper is 0.3111.
Why are they different? Is it a typo?

### Expected behavior

0.3111

### Actual behavior

0.30966

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

```python
# Put your Python code snippet here.
```

### PR DESCRIPTION
**Pull request by pllim**
### Description

This pull request is to clarify `Om0` value discrepancy between Planck18 and the associated paper.

@lewtonstein , does this make thing clearer? If not, please provide suggestion. Also, please note that `cosmology.Planck18_arXiv_v2` will be replaced with `cosmology.Planck18` from `astropy` 4.2 onwards (#10915).

Fixes #10957

### CODE REVIEW

File: docs/cosmology/index.rst [hunk 1]
```diff
@@ -305,6 +305,13 @@ Planck15  Planck Collab 2015, Paper XIII 67.7  0.307 Yes
 Planck18  Planck Collab 2018, Paper VI   67.7  0.310 Yes
 ========  ============================== ====  ===== =======
 
+.. note::
+
+  Unlike the Planck 2015 paper, the Planck 2018 paper included massive
```
Comment (aconley) [inline docs/cosmology/index.rst:310]: Nit: included -> includes

File: docs/cosmology/index.rst [hunk 2]
```diff
@@ -305,6 +305,13 @@ Planck15  Planck Collab 2015, Paper XIII 67.7  0.307 Yes
...
+  neutrinos in ``Om0`` but Planck18 here included them in ``m_nu`` instead
```
Comment (aconley) [inline docs/cosmology/index.rst:311]: Nit: but the Planck18 object includes them in...

File: docs/cosmology/index.rst [hunk 3]
```diff
@@ -305,6 +305,13 @@ Planck15  Planck Collab 2015, Paper XIII 67.7  0.307 Yes
...
+  for consistency. Hence, the ``Om0`` value in Planck18 differs slightly
+  from the Planck 2018 paper.
```
Comment (aconley) [inline docs/cosmology/index.rst:313]: but represents the same cosmological model.

**Comment by pllim** [comment]
Thanks, @aconley ! How does it look now?

**Review by aconley — APPROVED** [review]

**Review by bsipocz — APPROVED** [review]

**Comment by bsipocz** [comment]
Thank you

**Pull request merged**